src/utilities/util_main.c

```diff
@@ -284,6 +291,11 @@ main(int argc, char *argv[])
         (void)util_err(NULL, ret, NULL);
         goto err;
     }
+
+    /* If we only want to verify the metadata, that is done in wiredtiger_open. We're done. */
+    if (func == NULL && meta_verify)
+        goto done;
+
```

Comment:
The comment above this line `/* Open the database and a session. */` probably needs to be split up since this line is in the middle (or removed entirely, it's not really adding a lot of value).

Reply:
Fixed.